onnxruntime/contrib_ops/rocm/bert/transformer_common.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/rocm/rocm_common.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace rocm {
+
+class TransformerOptions {
+ public:
+  static const TransformerOptions* GetInstance();
+
+  bool IsPrecisionMode() const { return is_precision_mode_; }
+
+  bool DisablePersistentSoftmax() const { return disable_persistent_softmax_; }
+
+  bool DisableHalf2() const { return disable_half2_; }
+
+  bool IsTuningEnabled() const { return tuning_; }
+
+  void Initialize(int value) {
+    is_precision_mode_ = (value & 0x01) > 0;
+    disable_persistent_softmax_ = (value & 0x02) > 0;
+    disable_half2_ = (value & 0x04) > 0;
+    tuning_ = (value & 0x08) > 0;
```

Comment:
This is why we should use an API instead of env ;)

Reply:
Yes :)